conda_forge_tick/migrators/core.py

```diff
@@ -287,6 +287,19 @@ def parse_already_pred() -> bool:
         if bad_attr:
             LOGGER.debug("%s: bnad attr" % __name)
 
+        # Check that all upstreams are installable
+        arch_packages = {}
+        # TODO: global variable this somewhere and add aarch and ppc
```

Comment:
Full list is actually:

```py
    platforms = [
        "noarch",
        "linux-64",
        "osx-64",
        "win-32",
        "win-64",
        "linux-armv7l",
        "linux-ppc64le",
        "linux-aarch64",
    ]
```

FWIW

Reply:
That's what I needed (although mamba seems to know to add noarch to everything)

Reply:
Also do we build win-32 anymore?

Reply:
We do not build win-32 iiuic.

Reply:
Also how does this interact with selectors? (We don't have linux-64 as a selector, we have just linux).

Reply:
Also do we build `armv7l`?